tools/.lintstagedrc.json

```diff
@@ -1,15 +1,9 @@
 {
-  
   "../**/*.{md,java,js,sh,yaml,yml}|*.{md,java,js,sh,yaml,yml}": [
     "prettier --write"
   ],
-  "../**/*.dart|*.dart": [
-    "flutter format"
-  ],
-  "../client/**": [
-    "./verify-client.sh"
-  ],
-  "../server/**": [
-    "./verify-server.sh"
-  ]
+  "../**/*.dart|*.dart": ["flutter format"],
```

Comment:
Did you run prettier on this file? I'm fine with removing the empty line 2 but I think the rest of it works better with the original layout.